blog/se1/index.qmd

```diff
@@ -0,0 +1,28 @@
+---
+author: Caleb Kendra, Alish Chhetri, and Evan Nelson
+title: Programming Over Time
+date: '2023-11-29'
+date-format: YYYY
+categories: [post, developer development, software engineering]
+description: <em>This is our first blog about the `Software Engineering at Google` book!</em>
+page-layout: full
+---
+
+# Summary
+
+In the preface of "Software Engineering at Google" in the Programming Over Time section we are first introduced to the idea that "Software Engineering" is not just the act of creating code, but also the creation of processes that allow us to create maintainable and sustainable software processes. In this sense, when we release a version of the software we created, we don't want that software to be the final version but rather a building block upon which we can improve. In order to succeed in this goal of sustainability and growth, the book emphasizes three points to be thinking of while creating software:
+
+- Time - In order to make sure that software is created with future improvements in mind, we need to plan out how the software we create will change from the start and finish of a certain project. This means we need to open up our code to make sure that it can be improved upon in future releases.
```

Comment:
Again, do not soft line-wrap. Also, do not use a single `-`. Instead you should use a colon here. Also, you should probably put the word "time" in bold using the correct feature of Markdown. Is what you are saying true --- can you really know the "finish" of a project?